.gitignore

```diff
@@ -26,7 +26,7 @@ platforms/
 plugins/
 plugins/android.json
```

Comment:
True, not sure why it's there, it just got generated with the ionic project. I'll remove it.